index.html

```diff
@@ -219,16 +223,16 @@ <h1 id="h1-title">Loading...</h1>
 				$element.find('Size').text(),
 				name
 			);
-			
+
 			if (!fileOrFolder.hidden) {
 				// crude method to force addition of folders
 				var folderName = fileOrFolder.folderName;
-				if (folderName && folderName != '' && folderName != "__ROOT__" && !createdFolders[folderName]) {
+				if (folderName && folderName != '' && folderName != "__ROOT__" && createdFolders[folderName]) {
```

Comment:
Thanks for the adjustments

Wouldn't this code cause it to duplicate folder entries?